bin/fix-dbs-parentag.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+from __future__ import print_function, division
+from pprint import pprint
+from collections import defaultdict
+from dbs.apis.dbsClient import DbsApi
+
+endpoint = 'https://cmsweb.cern.ch/dbs/prod/global/DBSReader'
+dbs = DbsApi('https://cmsweb.cern.ch/dbs/prod/global/DBSReader')
+
+def listOfParentDatasets(dataset):
+    parents = dbs.listDatasetParents(dataset=dataset)
+    parentDatasets = []
+    for parent in parents:
+        parentDatasets.append(parent['parent_dataset'])
+    return parentDatasets
+
+def listBlocksWithNoParents(dataset):
+    allBlocks = dbs.listBlocks(dataset=dataset)
+    blockNames = []
+    for block in allBlocks:
+        blockNames.append(block['block_name'])
+    parentBlocks = dbs.listBlockParents(block_name=blockNames)
```

Comment:
Does it accept a list of block names? Or just a string with a single block name?

Reply:
only single block name